### DISCUSSION TITLE
Building an Akash Delegation Dashboard

### DISCUSSION THREAD

**Discussion by ruslan-tsybenov** [akash-network/community#280] [Ideas]
I am propose the development of a dashboard for monitoring Delegations. In addition, we are proud to announce that the entire codebase for our analytics solutions will be open-source. 

**Features and Functionality**

The proposed dashboard will have the following features:
_Validator Metrics Monitoring_:  Dashboard’s function will track metrics such as:
validator commission, sum of self-bonded and delegated tokens, voting power, uptime, and participation in governance proposals.
_Contribution tracking_: We will track every contribution of validators like developments or posts on social media. 
_Validator comparing feature_: The dashboard performs the actual comparison based on the selected criteria. The dashboard will present data in a visual format, such as charts, graphs, tables, or other visual representations. The visualizations will be designed to make the data easy to understand and interpret.

**Conclusion**
Our team already provides such solutions, and we have full confidence in our ability to promptly and efficiently implement our proposed strategies. We believe that this development is crucial for the success of the Akash Network.
The main idea of the proposal is to make the process of distribution of delegation clear and fair.

**Comment by hewi333** [comment]
Hi Cybenovr! Thanks for bringing forth this Dashboard idea. https://www.mintscan.io/akash/validators has a pretty robust summary of validators on the Akash Network but doesn't include a _Contribution tracking_ section that you've called out. You said your team already provides such solutions, can you provide some examples or past work?
